app/script/localization/translations/webapp-fr.js

```diff
@@ -478,6 +478,9 @@ z.string.fr.preferencesOptionsNotificationsNone = 'Désactiver';
 z.string.fr.preferencesOptionsNotificationsObfuscate = 'Masquer les détails';
 z.string.fr.preferencesOptionsNotificationsObfuscateMessage = 'Afficher l’expéditeur';
 z.string.fr.preferencesOptionsNotificationsOn = 'Afficher l’expéditeur et le message';
+z.string.fr.preferencesOptionsCallLogs = 'Dépannage';
+z.string.fr.preferencesOptionsCallLogsGet = 'Sauvegarder le rapport de débogage des appels';
+z.string.fr.preferencesOptionsCallLogsDetail = 'Cette information aide l’assistance de Wire à diagnostiquer les problèmes d'appel.';
```

Comment:
`d'appel` -> `d’appel`

There is the wrong apostrophe in the french translations.. and that's why the `;`  look red in the preview (lines 485-486)

Reply:
Yes, that was already fixed.